src/content/reference/react/use-client.md

```diff
@@ -215,39 +214,39 @@ This means that the component definition for `FancyText` will both be evaluated
 
-To clarify, `'use client'` defines the boundary between server and client code on the _module dependency tree_, not the render tree.
+Recall that `'use client'` defines the boundary between server and client code on the _module dependency tree_, not the render tree.
 
 <Diagram name="use_client_module_dependency" height={200} width={500} alt="A tree graph with the top node representing the module 'App.js'. 'App.js' has three children: 'Copyright.js', 'FancyText.js', and 'InspirationGenerator.js'. 'InspirationGenerator.js' has two children: 'FancyText.js' and 'inspirations.js'. The nodes under and including 'InspirationGenerator.js' have a yellow background color to signify that this sub-graph is client-rendered due to the 'use client' directive in 'InspirationGenerator.js'.">
 `'use client'` defines the boundary between server and client code on the module dependency tree.
 </Diagram>
 
 In the module dependency tree, we see that `App.js` imports and calls `Copyright` from the `Copyright.js` module. As `Copyright.js` does not contain a `'use client'` directive, the component usage is rendered on the server. `App` is rendered on the server as it is the root component.
 
-Client Components can render Server Components because you can pass JSX as props. In this case, `InspirationGenerator` receives `Copyright` as [children](/learn/passing-props-to-a-component#passing-jsx-as-children). However, the `InspirationGenerator` module never directly imports the `Copyright` module nor calls the component, all of that is done by `App`.
+Client Components can render Server Components because you can pass JSX as props. In this case, `InspirationGenerator` receives `Copyright` as [children](/learn/passing-props-to-a-component#passing-jsx-as-children). However, the `InspirationGenerator` module never directly imports the `Copyright` module nor calls the component, all of that is done by `App`. In fact, the `Copyright` component is fully executed before `InspirationGenerator` starts rendering.
 
 The takeaway is that a parent-child render relationship between components does not guarantee the same render environment.
 
 </DeepDive>
 
 ### When to use `'use client'` {/*when-to-use-use-client*/}
 
-With `'use client'`, you can determine what component usages will be Client Components. As Server Components are default, here is a brief overview of the advantages and limitations to Server Components to determine when you need to mark something as client rendered.
+With `'use client'`, you can determine when components are Client Components. As Server Components are default, here is a brief overview of the advantages and limitations to Server Components to determine when you need to mark something as client rendered.
 
 For simplicity, we talk about Server Components, but the same principles apply to all code in your app that is server run.
 
-#### Advantages {/*advantages*/}
+#### Advantages of Server Components {/*advantages*/}
```

Comment:
I only made minor edits here but I don't love this section in its current form. I think of this less as advantages/disadvantages and more that in many cases they are for different things! more like

* unique capabilities of server components
* unique capabilities of client components
* when to prefer one or the other if you don't need either one

Reply:
We can re-title this and re-word to frame as capabilities!